multiple-entity-row.js

```diff
@@ -315,4 +315,4 @@
     }
 
     customElements.define('multiple-entity-row', MultipleEntityRow);
-})(window.LitElement || Object.getPrototypeOf(customElements.get('hui-view')));
+})(window.LitElement || Object.getPrototypeOf(customElements.get('hui-masonry-view')));
```

Comment:
Would this work to keep < 0.116 working?

```
customElements.define("multiple-entity-row", MultipleEntityRow);
})(
  window.LitElement || customElements.get("hui-masonry-view")
    ? Object.getPrototypeOf(customElements.get("hui-masonry-view"))
    : Object.getPrototypeOf(customElements.get("hui-view"))
);
```

Reply:
It should. Can be simplified with the following:
```suggestion
})(window.LitElement || Object.getPrototypeOf(customElements.get('hui-masonry-view')) || Object.getPrototypeOf(customElements.get('hui-view')));
```

Reply:
The solution by @firstof9 works in HA <0.116 and 0.116

@benct This gives an error in HA <0.116 (but works in 0.116):
```
Uncaught TypeError: Cannot convert undefined or null to object
    at Function.getPrototypeOf (<anonymous>)
    at multiple-entity-row.js:318
```

Reply:
Sorry, try this instead..
```suggestion
})(window.LitElement || Object.getPrototypeOf(customElements.get('hui-masonry-view') || customElements.get('hui-view')));
```

Reply:
Now it's working fine for 0.115 & 0.116